### PR TITLE
Add machine-healthcheck scenarios

### PR DESCRIPTION
**Pull request by jhou1**
The test verifies a machine is remediated when it has the unhealthyCondition defined in a mhc resource.

### CODE REVIEW

**Comment by jhou1** [comment]
@akostadinov @pruan-rht Would you please review? Thank you

File: features/machine/machine_healthcheck.feature [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    Given I pick a random machineset to scale
+    # Create MHC
+    When I run oc create over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/cloud/mhc/mhc1.yaml" replacing paths:
+      | n                                                                                  | openshift-machine-api       |
+      | ["metadata"]["name"]                                                               | mhc-<%= machine_set.name %> |
+      | ["spec"]["selector"]["matchLabels"]["machine.openshift.io/cluster-api-cluster"]    | <%= machine_set.cluster %>  |
+      | ["spec"]["selector"]["matchLabels"]["machine.openshift.io/cluster-api-machineset"] | <%= machine_set.name %>     |
+    Then the step should succeed
+    Given I register clean-up steps:
```
Comment (pruan-rht) [inline features/machine/machine_healthcheck.feature]: @jhou1 perhaps use this existing step instead?  `Given I ensure "mch-#{machine_set.name}" machinehealthcheck is deleted`?

Comment (jhou1) [reply features/machine/machine_healthcheck.feature]: @pruan-rht Thanks for the comment, I updated it now.

**Pull request merged**